aixplain/__init__.py

```diff
@@ -36,6 +37,16 @@
 # Validate API keys
 from aixplain.utils import config
 
+if config.AIXPLAIN_ENV == "dev":
+    config.BACKEND_URL = "https://dev-platform-api.aixplain.com"
+    config.MODELS_RUN_URL = "https://dev-models.aixplain.com/api/v1/execute"
+elif config.AIXPLAIN_ENV == "test":
+    config.BACKEND_URL = "https://test-platform-api.aixplain.com"
+    config.MODELS_RUN_URL = "https://test-models.aixplain.com/api/v1/execute"
+else:
+    config.BACKEND_URL = "https://platform-api.aixplain.com"
+    config.MODELS_RUN_URL = "https://models.aixplain.com/api/v1/execute"
+
 if config.TEAM_API_KEY == "" and config.AIXPLAIN_API_KEY == "":
```

Comment:
@lucas-aixplain code cannot determine environment. This work is not compliant with the 12 factor app spec.